code/obj/item/gun/ammo.dm

```diff
@@ -1060,6 +1060,12 @@
 	cycle = 0
 	recharge_rate = 10.0
 
+/obj/item/ammo/power_cell/self_charging/howitzer
+	name = "Miniaturized SMES"
+	desc = "This thing is huge! How did you even lift it put it into the gun?"
+	charge = 2500.0
+	max_charge = 2500.0
```

Comment:
Now I want this to just have a regular SMES sprite, haha.

Reply:
Having more funny and varied power cell sprites would be good. But I'm just gonna merge this as is.